Image.js

```diff
@@ -29,9 +29,9 @@ module.exports = {
     'use strict';
     // Select random image provider
     if (this.currentProvider==='random') {
-      var l=Object.keys(this.providers);
-      var i=Math.floor(Math.random()*l.length);
-      return l[i];
+      var providersList=Object.keys(this.providers);
```

Comment:
```suggestion
      var providerNames=Object.keys(this.providers);
```
We want to avoid data types in names, like `list`.